circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -26,7 +26,7 @@
 from qiskit.quantum_info import PauliList
 
 from ..utils.observable_grouping import observables_restricted_to_subsystem
-from ..utils.transforms import separate_circuit
+from ..utils.transforms import separate_circuit, _partition_labels_from_circuit
```

Comment:
Maybe we should expose this function in another PR as well?

Reply:
Since we're in a time crunch, we can push this through and make that function public next release. I'd be fine with that too. Up to you